templates/500.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+{% block title %}500 - Whoops!{% endblock %}
+{% block content %}
+  <div id="center">
+    <h1 class="red">500 - Whoops!</h1>
+    <h2>Something that should have gone right went wrong</h2>
+    <h2><a href="/">Try again from the start</a>, or reach us at the following email address:</h2>
+    <div id="code-box">
+      <code>muncssociety at gmail dot com</code>
+    </div>
+
+    {% if sentry_event_id %}
+      <script>
+      Sentry.init({ dsn: 'https://{{ sentry_dsn }}@sentry.io/<project>' });
```

Comment:
The DSN in the environment variable is the whole part, including the protocol, domain name, etc.

```suggestion
      Sentry.init({ dsn: '{{ sentry_dsn }}' });
```

Reply:
This time you can just push a button, you don't need to actually make the changes yourself.